sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -453,6 +453,13 @@ def test_series_getitem(self):
     s.index = s.index.map(float)
     self._run_test(lambda s: s[1.5:6], s)
 
+  def test_series_is_unique(self):
+    s_unique = pd.Series(range(10))
+    s_dupes = pd.Series(i % 4 for i in range(10))
```

Comment:
A stronger test would be range(100) + [0].

Reply:
Done, thanks